docs/source/extensions_tutorial/extensions_tutorial_home.rst

```diff
@@ -0,0 +1,17 @@
+Extending NWB
+=============
+
+Neurophysiology is always changing as new technologies are developed. While the core NWB schema supports many of the
+most common data types in neurophysiology, we need a way to accommodate new technologies and unique metadata needs.
+Neurodata extensions (NDX) allow us to  define new data types. These data types can extend core types, contain core
+types, or can be entirely new. These extensions are formally defined with a collection of YAML files as defined by
+the `NWB Specification Language <https://schema-language.readthedocs.io/en/latest/index.html>`_.
+
+.. toctree::
+    :maxdepth: 2
+
+    2_create_extension_spec_walkthrough
+    3_spec_api
+    4_auto_api
+    5_custom_api
+    6_documenting_extension
```

Comment:
@bendichter thanks for cleaning up the extension docs!

It would be great to also add a section ``7_sharing_extensions`` to discuss publishing the extension in the catalog. Here we can focus on the actual process of sharing extensions but we should also link to the docs on the NDX Catalog for further details, e.g.:

- https://nwb-extensions.github.io/proposal_review
- https://nwb-extensions.github.io/sharing_strategies
- https://nwb-extensions.github.io/sharing_guidelines

Reply:
@bendichter sorry, I realize the term "cleaning up" is a bit of an understatement its really more of an overhaul and expansion. In any case, nice work!

Reply:
@oruebel I agree, but let's put that on the "to do" list. I think Ryan would be the best person to write that section.